common/src/main/java/com/wynntils/screens/maps/AbstractMapScreen.java

```diff
@@ -368,16 +369,21 @@ protected void centerMapAroundPlayer() {
 
     protected void centerMap() {
         updateMapCenter(MAP_CENTER_X, MAP_CENTER_Z);
-        setZoom(0);
+        adjustZoomStep(0);
```

Comment:
FIXME: This is not correct. There should be a setZoomStep method, and an adjustZoomStepByDelta method.